scripts/simplify_network.py

```diff
@@ -391,10 +391,11 @@ def busmap_by_stubs(network, matching_attrs=None):
     def attrs_match(u, v):
         return (
             matching_attrs is None
+            # TODO account for a group length of more than just one pair
             or (
                 network.buses.loc[u, matching_attrs]
                 == network.buses.loc[v, matching_attrs]
-            ).all()
```

Comment:
Do we need this change?
I feel like if we include this change, in the case of multiple inputs we loose the original flow

Reply:
Have revised to make it work both for a single and for a multiple match.

Reply:
Perfect :)